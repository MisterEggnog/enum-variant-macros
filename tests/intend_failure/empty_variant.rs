use enum_variant_macros::*;
#[derive(FromVariants)]
enum Variants {
	Int(i32),
	Float(f32),
	None,
}
fn main() {
}
