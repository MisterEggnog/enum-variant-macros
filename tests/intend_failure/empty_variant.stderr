error: FromVariants can only use unamed members, failed for Variants::None
 --> tests/intend_failure/empty_variant.rs:2:10
  |
2 | #[derive(FromVariants)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `FromVariants` (in Nightly builds, run with -Z macro-backtrace for more info)
